Add another IDE Setup/Configuration tutorial

PBfordev opened the pull request:
I took a look at some other tutorials.

**Section IDE Setup/Configuration**
_Programming C++ GUIs with the wxWidgets Library_
Not only it is rather old (published in 2006), I think setup/configuration are not its main focus.

_Intellisense and wxWidgets_ 
The link does not work, moreover, I would expect Intellisense work out of the box these days.

**Section wxWidgets and Others**
_Using wxWidgets with OpenCV_ 
Looks rather obsolete. I think even my [wxOpenCVTest](https://github.com/PBfordev/wxopencvtest) could be more useful nowadays; however, it has just code.

tierra commented on docs/tutorials/index.md:
```diff
@@ -55,6 +55,20 @@ page on the community wiki.
 ## IDE Setup/Configuration
 
 <div class="list-group my-3">
+<li class="list-group-item">
```
minor nit: indentation seems off here

PBfordev replied:
Thanks, fixed

tierra commented:
I think "Programming C++ GUIs with the wxWidgets Library" could be moved up one section more appropriately. You're right, it's more of a Getting Started guide.

For setup/configuration, we never bothered linking to all the [guides in the wxWiki](https://wiki.wxwidgets.org/Install), and many of them have certainly become fairly outdated (mostly in terms of versions of IDEs and compilers, it's not like wx has changed much itself). Regardless, it's a huge list of IDE-specific guides in many different environments, and is probably the second most valuable resource for those guides (after the official platform-specific docs in the library itself). It's likely worth at least linking to that top level list of guides here, next to your new guide here.

Go ahead and remove the Intellisense link, it does look dead to me.

PBfordev commented:
As suggested by tierra, I have changed sections for "Programming C++ GUIs with the wxWidgets Library" tutorial and removed the non-working Intellisense one.

As for linking wxwiki's installer guides. I am not sure how to best (HTML-wise) add the link to the section. Perhaps the generic Guides & Tutorials link on the top of the page could be enough?

BTW, I added my tutorial based on vadz's suggestion in wx-users, not out of my hubris.

tierra commented:
No problem on wiki guides, this is great as is.
